core/server/master/src/main/java/alluxio/master/file/FileSystemMasterJobServiceHandler.java

```diff
@@ -54,9 +54,9 @@ public void getFileInfo(GetFileInfoPRequest request,
     GetFileInfoPOptions options = request.getOptions();
 
     RpcUtils.call(LOG,
-        (RpcUtils.RpcCallableThrowsIOException<GetFileInfoPResponse>) () -> GetFileInfoPResponse
-            .newBuilder().setFileInfo(GrpcUtils.toProto(mFileSystemMaster.getFileInfo(fileId)))
-            .build(),
+            () -> GetFileInfoPResponse
```

Comment:
spacing issues?

Reply:
It's just the casting '() -> {...}' to 'RpcUtils.RpcCallableThrowsIOException<GetFileInfoPResponse>' is redundant.